src/opencast.js

```diff
@@ -544,25 +544,17 @@ export class Opencast {
     }
 
     // Prepare template "view": the values that can be used within the template.
-    const ltiCourseId = this.#ltiSession?.context_id;
-    const roleOAuthUser = this.#currentUser.roles.find(r => r === 'ROLE_OAUTH_USER');
-
-    let defaultReadRoles = [this.#currentUser.userRole];
-    let defaultWriteRoles = [this.#currentUser.userRole];
-    if (ltiCourseId) {
-      defaultReadRoles.push(`${ltiCourseId}_Learner`, `${ltiCourseId}_Instructor`);
-      defaultWriteRoles.push(`${ltiCourseId}_Instructor`);
-    }
-
     const view = {
-      userName: escapeString(this.#currentUser.user.username),
-      userRole: escapeString(this.#currentUser.userRole),
-      roleOAuthUser: escapeString(roleOAuthUser),
-      ltiCourseId: escapeString(ltiCourseId),
-      defaultReadRoles: defaultReadRoles.map(r => escapeString(r)),
-      defaultWriteRoles: defaultWriteRoles.map(r => escapeString(r)),
+      user: this.#currentUser,
+      lti: this.#ltiSession,
+      roleOAuthUser: this.#currentUser.roles.find(r => r === 'ROLE_OAUTH_USER'),
     };
-    return Mustache.render(template, view);
+
+    const originalEscape = Mustache.escape;
+    Mustache.escape = escapeString;
+    const out = Mustache.render(template, view);
+    Mustache.escape = originalEscape;
+    return out;
```

Comment:
Since this happens twice now, after #672, and since we found out that there is no nice way within Mustache, to do this, I could see a utility function here. But no hard opinion, I guess. :thinking: